### DISCUSSION TITLE
Question regarding downloading reblogs

### DISCUSSION THREAD

**Discussion by mtae** [TumblThreeApp/TumblThree#367] [Q&A (Question & Answer)]
Is there an option to download only reblogs of posts from that blog?
i.e. exclude only reblogs that have a source value that is different from the blog being downloaded?

**Comment by thomas694** [comment]
There is no option to download only reblogged posts.

**Comment by mtae** [comment]
Sorry I should clarify, I meant with the "download originals" option, is it possible to include self-reblogs? 
Say I'm downloading the blog tumblr323 and they reblog their own original post and add content to it, would it be included?

**Comment by thomas694** [comment]
With the option "download originals" (checkbox "Download reblogged posts" off) reblogged posts of their old posts to their own blog are skipped, even if additional content was added. At the moment it doesn't recognize this additional content and only distinguishes between normal and reblogged posts.

**Comment by thomas694** [comment]
A blog's own reblogged posts are now treated as valid content independent from the option "download reblogged posts".

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
